Source/IGListAdapterUpdater.m

```diff
@@ -84,6 +84,7 @@ - (void)performReloadDataWithCollectionView:(UICollectionView *)collectionView {
 
     [delegate listAdapterUpdater:self willReloadDataWithCollectionView:collectionView];
     [collectionView reloadData];
+    [collectionView.collectionViewLayout invalidateLayout];
     [collectionView layoutIfNeeded];
```

Comment:
because of the layout invalidation, i don't think we need `-layoutIfNeeded` anymore. want to check?